src/Modules/Authentication/components/ChangePassword/ChangePassword.tsx

```diff
@@ -1,6 +1,180 @@
+            Old Password
+          </label>
+          <div className="relative">
+            {/* Password Input */}
+            <input
+              className="placeholder:font-nunito placeholder:text-slate-400 block w-full bg-transparent  border-2 border-slate-300 rounded-md py-2 pl-10 pr-10 shadow-sm focus:outline-none focus:border-sky-500 focus:ring-sky-500 focus:ring-1 sm:text-sm md:text-base lg:text-lg"
+              placeholder="Type your old password"
+              type={showOldPassword ? "text" : "password"}
+              {...register("password", PasswordValidation)}
+            />
+
+            {/* Lock Icon */}
+            <span className="absolute inset-y-0 left-0 flex items-center pl-3">
+              <span className="sr-only">Old Password Icon</span>
+              <i className="fa-solid fa-lock text-white"></i>
+            </span>
+
+            {/* Toggle Visibility Button */}
+            <button
+              onMouseDown={(e) => e.preventDefault()}
+              onMouseUp={(e) => e.preventDefault()}
+              type="button"
+              onClick={() =>
+                toggleVisibility(setShowOldPassword(!showOldPassword))
+              }
+              className="absolute inset-y-0 right-0 flex items-center pr-3"
+            >
+              <span className="sr-only">
+                {showOldPassword ? "Hide password" : "Show password"}
+              </span>
+              <i
+                className={`text-white ${
+                  showOldPassword ? "fa-solid fa-eye" : "fa-solid fa-eye-slash"
+                }`}
+              ></i>
+            </button>
+
+            {/* Password Error Message */}
+            {errors.password && (
+              <span className="text-red-500 absolute text-xs md:text-sm lg:text-base mt-1">
+                {errors.password.message}
+              </span>
+            )}
+          </div>
+        </div>
+
+        {/* New Password Input Group */}
+        <div className="input-group mt-8  my-6 lg:my-14">
+          <label className="block mb-2 text-sm md:text-base lg:text-lg">
+            New Password
+          </label>
+          <div className="relative">
+            {/* Password Input */}
+            <input
+              className="placeholder:font-nunito placeholder:text-slate-400 block w-full bg-transparent  border-2 border-slate-300 rounded-md py-2 pl-10 pr-10 shadow-sm focus:outline-none focus:border-sky-500 focus:ring-sky-500 focus:ring-1 sm:text-sm md:text-base lg:text-lg"
+              placeholder="Type your New Password"
+              type={showPassword ? "text" : "password"}
+              {...register("password_new", {
+                validate: (value) =>
+                  value === getValues("password") || "password dont match",
+              })}
+            />
+
+            {/* Lock Icon */}
+            <span className="absolute inset-y-0 left-0 flex items-center pl-3">
```

Comment:
I suggest reading about logically properties. It is a good practice to use them when possible. For example, if we want to make this element right to left. We will need to add more classes to handle that.
If we use the logical properties for the right like inset-inlin-end, this will make the button at the end of the text direction (at the right in ltr langauge, at left in rtl languages). We have logical properties for padding and margin as well.